fuzz_lightyear/supplements/factory.py

```diff
@@ -19,9 +20,17 @@
 from fuzz_lightyear.util import listify_decorator_args
 
 
-def register_factory(keys: Union[str, Iterable[str]]) -> Callable:
+def _none_func() -> None:
```

Comment:
```suggestion
def _returns_none() -> None:
```

This function name is much more self-descriptive.